pontoon/gpt/openai_connection.py

```diff
@@ -0,0 +1,51 @@
+from openai import OpenAI
+
+
+class OpenAIService:
+    def __init__(self):
+        self.client = OpenAI()
+
+    def get_translation(
+        self, english_text, translated_text, characteristic, target_locale_name
+    ):
+        system_messages = {
```

Comment:
I find this hard to read. Please declare each of the three characteristics as variables and then use them like this:

```python
system_messages = {
    "informal": informal,
    "formal": formal,
    "alternative": alternative,
}
```